modules/tibc/core/exported/client.go

```diff
@@ -20,6 +20,8 @@ const (
 	// BSC is the client type for a bianance smart chain client.
 	BSC string = "008-bsc"
 
+	ETH string = "009-eth"
+
 	// Fabric is the client type for a hyperledge fabric client.
 	Fabric string = "009-fabric"
```

Comment:
ETH string = "009-eth" 
Fabric string = "009-fabric"
Is it conflicting?

Reply:
ETH string = "010-eth"